requirements.txt

```diff
@@ -1,3 +1,5 @@
-numpy~=1.21.2
+numpy~=1.19.2
 scipy~=1.7.1
-matplotlib~=3.4.3
\ No newline at end of file
+matplotlib~=3.4.3
+tensorflow~=2.6.0
```

Comment:
Trzeba rozważyć przejście na 2.7.0, Colab podstawowo działa na tej wersji (instalowanie nie jest tak upierdliwe, ale GPU zdaje się nie działać na 2.6.0, a przynajmniej nie udało się nam uruchomić na tej wersji)

Reply:
Jak GPU nie działa na 2.6.0 oraz Colab podstawowo działa na 2.7.0 to możemy przejść na tą wersję.